docs/explanation/logical-axiomatization.md

```diff
@@ -0,0 +1,101 @@
+# Logical axiomatization of classes & use of reasoning
+
+This explainer requires understanding of ontology classifications, please see "an ontology as a classification" section of the [introduction to ontologies documentation](../explanation/intro-to-ontologies.md) if you are unfamiliar with these concepts. 
+
+## What are logical axioms
+
+Logical axioms are relational information about a class that are primarily aimed at machines (as opposed to annotations like textual definitions which are primarily aimed at humans). These logical axioms allow reasoners to assist in and verify classification, lessening the development burden and enabling expressive queries.
+
+## What should you axiomatize?
+
+Ideally, everything in the definition should be axiomatize (though this is not always possible). For example, if we consider the cell type `oxytocin receptor sst GABAergic cortical interneuron` which has the textual definition: 
```

Comment:
Ideally, everything in the definition should be axiomatized when possible. For example, if we consider the cell type `oxytocin receptor sst GABAergic cortical interneuron`, which has the textual definition: